docs/src/main/sphinx/installation/plugins.md

```diff
@@ -0,0 +1,284 @@
+
+Trino uses a plugin architecture to extend its capabilities and integrate with
+various data sources and other systems. Trino includes many plugins as part of
+the binary packages - specifically the `tar.gz` archive and the Docker
+container.
+
+The main plugins implement some the following functionalities:
+
+* [Connectors](/connector)
+* [Access control systems](security-access-control)
+* [Event listeners](admin-event-listeners)
+* Additional [types](/develop/types) and [functions](/develop/functions)
+
+All plugins are optional for your use of Trino since they support a specific
+functionality, that is potentially not needed for your use. Plugins are located
+in the `plugin` folder of your Trino installation, and are loaded automatically
+during Trino startup.
+
+(plugins-downloading)=
+## Downloading
+
+Typically downloading a plugin is not necessary, since Trino binaries include
+many plugins as part of the binary packages.
+
+Every Trino release publishes of each plugin as a ZIP archive to the [Maven
+Central Repository](https://central.sonatype.com/). Refer to [plugins-list] for
+details. The specific location is derived from the Maven coordinates of each
+plugin as defined in the `pom.xml` of the source code for the plugin.
+
+For example the PostgreSQL connector plugin can be found in the
+`plugin/trino-postgresql` directory, and the `pom.xml` file contains the
+following identifier section:
+
+```xml
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>470</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-postgresql</artifactId>
+    <packaging>trino-plugin</packaging>
+```
+
+The Maven coordinates are therefore `io.trino:trino-postgresql:470`  with
+version or `io.trino:trino-postgresql` without version. Use this term for a
+[search to locate the
+artifact](https://central.sonatype.com/search?q=io.trino%3Atrino-postgresql).
+
+The coordinates translate into a path to the ZIP archive on the Maven Central
+Repository. Use this URL to download the plugin.
+
+```
+https://repo1.maven.org/maven2/io/trino/trino-postgresql/470/trino-postgresql-470.zip
+```
+
+Availability of plugins from other projects and organizations varies widely, and
+can require building a plugin from source.
+
+(plugins-installing)=
+## Installing
+
+Trino and the installed plugins must use the same version of Trino to be fully
+compatible. For example, a Trino plugin compiled for Trino 470 can not be used
+with older or newer versions of Trino such as Trino 430 or Trino 490.
+
+This is specifically important when installing plugins from other projects,
+vendors, or your [custom development](plugins-developing).
+
+To install a plugin, simply extract the ZIP archive into a directory in the
+`plugin` directory of your Trino installation on all nodes of the cluster. The
+directory contains all necessary resources. After a restart of Trino the plugin
```

Comment:
Recommend leading with the verb and making it declarative: Restart Trino to...

Reply:
Fixed